NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(add_ws_to_parse_data)
```

Comment:
Yes, we usually don't want to export everything because we don't want to take responsibility that these items remain unchanged for a "long time". During development you can use `devtools::load_all()` (Cmd+Shift+L in RStudio), it loads all internal functions too.